samples/graph-activity-feed/csharp/ActivityFeedNotification/Helpers/ChatMessageHelper.cs

```diff
@@ -0,0 +1,210 @@
+﻿using AdaptiveCards;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Graph;
+using Microsoft.Graph.Auth;
+using Microsoft.Identity.Client;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using TabActivityFeed.Model;
+
+namespace TabActivityFeed.Helpers
+{
+    public class ChatMessageHelper
+    {
+        public readonly IConfiguration _configuration;
+        IChatMessageHostedContentsCollectionPage chatMessageHostedContentsCollectionPage = new ChatMessageHostedContentsCollectionPage();
+
+        public ChatMessageHelper(IConfiguration config)
+        {
+            _configuration = config;
+        }
+
+        public async Task<ChatMessage> CreateChatMessageForChannel(TaskInfo taskInfo)
+        {
+            GraphServiceClient graphClientChat = GetGraphClient();
+            var chatMessage = new ChatMessage
+            {
+                Subject = null,
+                Body = new ItemBody
+                {
+                    ContentType = BodyType.Html,
+                    Content = "New Deployment: " + taskInfo.DeployementTitle
+                },
+            };
+            chatMessage.HostedContents = chatMessageHostedContentsCollectionPage;
+            var channelMessage = await graphClientChat.Teams[_configuration["teamId"]].Channels[_configuration["channelId"]].Messages
```

Comment:
_configuration["teamId"]] and _configuration["channelId"] are not present in appsettings.json, Is it possible to fetch these dynamically?